core/admin/mailu/api/v1/user.py

```diff
@@ -194,7 +199,7 @@ def patch(self, email):
             return {'code': 404, 'message': f'User {email} cannot be found'}, 404
 
         if 'raw_password' in data:
-            user_found.set_password(data['raw_password'])
+            user_found.set_password(data['raw_password'], keep_only_session='')
```

Comment:
I don't think there should be an empty string passed to set_password.

Reply:
As in we don't want to nuke the open sessions? Or it should be something that is more explicit like [] ?

It's fairly common for existing sessions to be destroyed upon password change, it ensures that if an attacker has one it will be revoked.

Reply:
I concur it is fairly common to destroy all sessions when the user's password is changed. Isn't this even a best practice?

Reply:
Yes, sessions should be cleared. It was just confusing to me how this is done.
Having an empty string as argument didn't occur to me as "clear all sessions".

What about
a) having the method "set_password" only do what it's name says and clearing sessions explicitely where we want to do it.
b) having the method "set_password" _always_ clear all sessions unless the argument keep_sessions is set to True (which will keep all sessions) or a set() which will keep the sessions in the set.

I think this will be more clear then.

Reply:
Check [0e4c29c](https://github.com/Mailu/Mailu/pull/2901/commits/0e4c29c26957f65e4ba776349a05d07e01fa200d) out

Reply:
👍